examples/call_v_from_c/README.md

```diff
@@ -0,0 +1,32 @@
+## A simple example to show how to call a function written in v from c
+
+### Compile as a shared library
+
+#### On Linux:
+
+Step 1: Compile the v code to a shared library using ``v -shared -prod v_test_print.v``
```

Comment:
```suggestion
Step 1: Compile the v code to a shared library using ``v -shared v_test_print.v``
```

Reply:
It is better to focus on a single flag at a time.

Reply:
The `-prod` was needed, only because V would use `gcc` in this case. Forcing it explicitly is clearer, and can let people see how to adapt to other situations/compilers by just substituting gcc everywhere.

Reply:
Thanks for the information.

Correction done.